ec2setup/algorithms/UploadToCounty/UploadToPostgresCountiesZips.py

```diff
@@ -0,0 +1,90 @@
+import boto3
+import json
+import psycopg2
+import numpy as np
+import pandas as pd
+import csv
+
+
+class UploadToPostgres():
+    def __init__(self):
+        with open("postgres_info.json") as json_file:
+            postgres_info = json.load(json_file)
+
+        self.db_host = postgres_info["DB_HOST"]
+        self.table_county = "script_county"
+        self.table_zipcode = "script_zip_code"
+        self.postgres_db = postgres_info['POSTGRES_DB']
+        self.postgres_user = postgres_info['POSTGRES_USER']
+        self.postgres_password = postgres_info['POSTGRES_PASSWORD']
+
+
+    def createCountyUploadData(self, s3_path):
+        ''' cleans data before and returns dictionary before uploading to db'''
+
+        data = pd.read_csv(s3_path)
+        county_names = data['County'].unique()
+        # number of county entries in the dataset
+        county_session_count = data['County'].value_counts().tolist()
+
+        county_total_energy = []
+        county_peak_energy = []
+        county_zip_codes = []
+
+        # loops on the unique county names to gather corresponding data
+        for x in range(len(county_names)):
```

Comment:
This might be a personal preference (and not sure if this was previous written code), but I prefer using enumerate to loop through with indexes, and using idx as the naming convention instead of x. So instead something like: `for idx, item in enumerate(county_names):` otherwise looks good!! The file overall is still loading for me, but will post an update when it's done

Reply:
done